Gemfile

```
@@ -59,6 +59,8 @@ gem "will_paginate", "~> 4.0.0"
 
 gem "friendly_id", "~> 5.5.0"
 
+gem "scimitar"
```

> Scimitar uses [semantic versioning](https://semver.org/) so you can be confident that patch and minor version updates for features, bug fixes and/or security patches will not break your application.

:yellow_circle: Subsequently we should pin to the major version that we currently tested against:

```suggestion
gem "scimitar", "~> 2.11"
```